classes/phing/tasks/system/ForeachTask.php

```diff
@@ -63,6 +63,9 @@ class ForeachTask extends Task
     /** Delimiter that separates items in $list */
     private $delimiter = ',';
 
+    /** Delimiter that separates items in $list */
+    private $delta = 'delta';
```

Comment:
There's no setter for `$delta` (which will be needed to override the default value).

Reply:
And while we're at it, please document the new property in `docs/guide/source/en` as well.